joomla/joomla-cms
# [com_fields fields view] SQL error on ordering

New issue.

### Summary of Changes

Go to any fields view and order by "Status", you'll get an SQL error

![image](https://cloud.githubusercontent.com/assets/9630530/19950911/8d81cf0e-a152-11e6-935a-cf6511d5d229.png)

This happens because the table field iss `state`, not `published`
![image](https://cloud.githubusercontent.com/assets/9630530/19951156/03c23658-a154-11e6-9379-2ad7c0072914.png)

This PR solves that.

### Testing Instructions

Very simple test. Code review, or:
1. Confirm the issue
2. Apply patch
3. Confirm there is no issue now

### Documentation Changes Required

None.
